src/main/java/james/command/AddListCommand.java

```diff
@@ -0,0 +1,29 @@
+package james.command;
+
+import james.JamesException;
+import james.task.TaskList;
+import james.ui.UI;
+
+/**
+ * The command to list all tasks in the task list.
+ */
+public class AddListCommand extends Command {
```

Comment:
ListTasksCommand might be a more appropriate class name 👍🏻

Reply:
That is appropriate for a change thank you